test/tests/transfers/multiapp_high_order_variable_transfer/tests

```diff
@@ -0,0 +1,42 @@
+[Tests]
+   design = "MultiAppMeshFunctionTransfer.md"
+   issues = "#13446"
+
+  [L2_Lagrange]
+    type = 'Exodiff'
+    input = 'master_L2_Lagrange.i'
+    exodiff = 'master_L2_Lagrange_out.e master_L2_Lagrange_out_sub0.e'
+    requirement = "The system shall support transfer first order elemental variable"
```

Comment:
The system shall support mesh to mesh interpolation transfers of first order elemental variables.